config/jobs/kubernetes/sig-scalability/DRA/sig-scalability-periodic-dra.yaml

```diff
@@ -235,3 +235,80 @@ periodics:
     annotations:
       testgrid-dashboards: sig-scalability-dra
       testgrid-tab-name: azure-dra-with-workload-master-scalability-100
+
+  - name: ci-kubernetes-e2e-gce-100-node-dra-with-workload
+    cluster: k8s-infra-prow-build
+    tags:
+      - "perfDashPrefix: gce-dra-100Nodes-with-workload"
+      - "perfDashBuildsCount: 270"
+      - "perfDashJobType: performance"
+    interval: 4h
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-e2e-scalability-common: "true"
+      preset-e2e-scalability-periodics: "true"
+      preset-e2e-scalability-periodics-master: "true"
+    annotations:
+      testgrid-dashboards: sig-scalability-dra
+      testgrid-tab-name: gce-dra-with-workload-master-scalability-100
+    decorate: true
+    decoration_config:
+      timeout: 3h
+    extra_refs:
+      - org: kubernetes
+        repo: kubernetes
+        base_ref: master
+        path_alias: k8s.io/kubernetes
+      - org: kubernetes
+        repo: perf-tests
+        base_ref: master
+        path_alias: k8s.io/perf-tests
+    spec:
+      containers:
+        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250710-e96fecb3d6-master
+          command:
+            - runner.sh
+            - /workspace/scenarios/kubernetes_e2e.py
```

Comment:
We really need to stop writing new jobs using these, kubernetes_e2e.py has been deprecated for like 7 years now and receives zero active development.

https://github.com/kubernetes/test-infra/tree/master/scenarios#deprecation-notice

Reply:
Probably the kubetest2 kops jobs. On second thought I'll withdraw this for now, but the tech debt here is sad on what is otherwise the latest efforts.

There are multiple layers of unsupported, undeveloped tools being depended on here (kubernetes_e2e.py, kubetest instead of kubetest2, kube-up.sh ...), sigh.

Resolving.

Reply:
@BenTheElder if it is easy to move to latest efforts, I will be willing to try it.  Not a huge fan of building on to tech debt